armi/interfaces.py

```diff
@@ -431,11 +431,10 @@ def specifyInputs(cs) -> Dict[Union[str, settings.Setting], List[str]]:
         The files returned by an implementation should be those that one would want
         copied to a target location when cloning a Case or CaseSuite. These can be
         absolute paths, relative paths, or glob patterns that will be interpolated
-        relative to the input directory. Aboslute paths will not be copied anywhere.
+        relative to the input directory. Absolute paths will not be copied anywhere.
 
-        The returned dictionary should be keyed off of a descriptive string, or an
-        actual Setting object. If a Setting is used, then the source CaseSettings object
-        will be updated to the new file location.
+        The returned dictionary should be keyed off of a Setting object, so the source
```

Comment:
@jakehader can you confirm this looks good and that I don't need to update the annotation in the definition?

Reply:
I would add one more sentence that says something along the lines of - While the dictionary keys are recommended to. E setting objects, the name of the setting as a string (e.g., "shuffleLogic") is still interpreted. If the string name does not point to a valid setting then this will lead to a failure.